dev/travis/js/config.php

```diff
@@ -0,0 +1,360 @@
+<?php
```

Comment:
So, I've done some brief experiments with Travis CI and here is what seems to be working:

1. Instead of bundling `config.php` add the minimum installation as follows (it does not add too much time to the execution):
> `echo "Installing Magento"`
> `mysql -uroot -e 'CREATE DATABASE magento2;'`
> `php bin/magento setup:install -q --admin-user="admin" --admin-password="123123q" --admin-email="admin@example.com" --admin-firstname="John" --admin-lastname="Doe"`
2. You may do some experiments with the static content deployment settings to add a bit more performance. Here is what worked for me (may not be the best solution, just experiment):
> `echo "Deploying Static Content"`
> `php bin/magento setup:static-content:deploy -f -q -j=2 --no-css --no-less --no-images --no-fonts --no-misc --no-html-minify`